adhocracy4/follows/static/follows/FollowButton.jsx

```diff
@@ -1,77 +1,67 @@
+import django from 'django'
+import React, { useState, useEffect } from 'react'
+import Alert from '../../../static/Alert'
+
 const api = require('../../../static/api')
-const django = require('django')
-const React = require('react')
-const Alert = require('../../../static/Alert')
 
-class FollowButton extends React.Component {
-  constructor (props) {
-    super(props)
+export const FollowButton = (props) => {
+  const [followed, setFollowed] = useState(undefined)
+  const [alert, setAlert] = useState(null)
 
-    this.state = {
-      followed: undefined,
-      follows: 0,
-      alert: null
-    }
+  const removeAlert = () => {
+    setAlert(null)
   }
 
-  removeAlert () {
-    this.setState({
-      alert: null
-    })
-  }
+  const followDescriptionText = followed
+    ? django.gettext('Click to no longer be updated about this project via email.')
+    : django.gettext('Click to be updated about this project via email.')
```

Comment:
added a description text as alert is not used in mB anymore but still is in a4, defakt, diid and we should have an explanation for SR readers